### PR TITLE
Implement semantic-release and conventional commits

### PR DESCRIPTION
**Pull request by jlegrone**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,6 +2,9 @@
 [![Build Status](https://travis-ci.org/open-tux/ember-bulma.svg?branch=master)](https://travis-ci.org/open-tux/ember-bulma)
 [![Ember Observer Score](http://emberobserver.com/badges/ember-bulma.svg)](https://emberobserver.com/addons/ember-bulma)
 
+[![Commitizen friendly](https://img.shields.io/badge/commitizen-friendly-brightgreen.svg)](http://commitizen.github.io/cz-cli/)
```
Comment (alexdiliberto) [inline README.md]: @jlegrone 👍 Commitizen, awesome

File: commitizen.config.js [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+  {
+    value: 'chore',
+    name: 'chore:    Changes to the build process or auxiliary tools\n            and libraries such as documentation generation'
+  },
+  {
+    value: 'revert',
+    name: 'revert:   Revert to a commit'
+  }
+];
+
```
Comment (alexdiliberto) [inline commitizen.config.js]: @jlegrone I'm good with this for now, we may want something like `style` as well

Comment (jlegrone) [inline commitizen.config.js]: added style and perf 😄

Comment (jlegrone) [inline commitizen.config.js]: any default scopes you think would be useful?

**Comment by jlegrone** [comment]
I'd like to get Travis set up to run semantic-release before merging.  @alexdiliberto or someone else with publish permissions, can you switch to this feature branch and follow these instructions?  If you end up with a local diff please feel free to push those changes and I can squash them in.

https://github.com/semantic-release/semantic-release#setup

File: commitizen.config.js (hunk 2 above)
Comment (alexdiliberto) [inline commitizen.config.js]: @jlegrone yea scopes are a definitely the trickier bit to standardize IMO. I lean to something like the default ember resolver for example: `component:date-picker`, `route:users`, `service:auth` but that only works for isolated updates. Then for broader stuff like `ci`, `scss`, etc. it's harder unless enforcing strictly atomic commits without exception

**Comment by alexdiliberto** [comment]
@jlegrone @jbailey4 You have both been added as collaborators via NPM

**Pull request merged**